instance/tests/api/test_instance.py

```diff
@@ -28,9 +28,12 @@
 
 from instance import github
 from instance.models.instance import SingleVMOpenEdXInstance
-from instance.models.server import OpenStackServer
 from instance.tests.api.base import APITestCase
-from instance.tests.models.factories.instance import SingleVMOpenEdXInstanceFactory
+from instance.tests.models.factories.instance import (
+    ConfiguringSingleVMOpenEdXInstanceFactory, ConfigFailedSingleVMOpenEdXInstanceFactory,
+    ErrorSingleVMOpenEdXInstanceFactory, RunningSingleVMOpenEdXInstanceFactory,
+    SingleVMOpenEdXInstanceFactory, TerminatedSingleVMOpenEdXInstanceFactory, WaitingSingleVMOpenEdXInstanceFactory
```

Comment:
Nit: with a big list of long ugly imported names like this, it's nicer to put one name per line.